rrfs-test/validated_yamls/templates/obtype_config/msonet_winds_288.yaml

```diff
@@ -0,0 +1,513 @@
+             inflation variable:
+               name: ObsFunction/ObsErrorFactorPressureCheck
+               options:
+                 variable: windNorthward
+                 inflation factor: 4.0
+
+         # Bounds Check
+         - filter: Bounds Check
+           apply at iterations: 0,1
+           filter variables:
+           - name: windEastward
+           - name: windNorthward
+           minvalue: -130
+           maxvalue: 130
+           action:
+             name: reject
+
+         # Gross Error Check
+         - filter: Background Check
+           filter variables:
+           - name: windEastward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     &cgross    [ 5.0 ]
+               error_min:  &error_min [ 1.0 ]
+               error_max:  &error_max [ 5.0 ]
+               variable: windEastward
+           where:
+           - variable: QualityMarker/windEastward
+             is_not_in: 3
+           action:
+             name: reject
+
+         # Gross Error Check
+         - filter: Background Check
+           filter variables:
+           - name: windNorthward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     *cgross
+               error_min:  *error_min
+               error_max:  *error_max
+               variable: windNorthward
+           where:
+           - variable: QualityMarker/windNorthward
+             is_not_in: 3
+           action:
+             name: reject
+
+         # Gross Error Check: cgross*0.7 if QualityMarker=3
+         - filter: Background Check
+           filter variables:
+           - name: windEastward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     &cgross0_7 [ 3.5 ]
+               error_min:  *error_min
+               error_max:  *error_max
+               variable: windEastward
+           where:
+           - variable: QualityMarker/windEastward
+             is_in: 3
+           action:
+             name: reject
+
+         # Gross Error Check: cgross*0.7 if QualityMarker=3
+         - filter: Background Check
+           filter variables:
+           - name: windNorthward
+           function absolute threshold:
+           - name: ObsFunction/WindsSPDBCheck
+             options:
+               wndtype:    [ 288 ]
+               cgross:     *cgross0_7
+               error_min:  *error_min
+               error_max:  *error_max
+               variable: windNorthward
+           where:
+           - variable: QualityMarker/windNorthward
+             is_in: 3
+           action:
+             name: reject
+
+         - filter: GOMsaver
+           filename: ./data/geovals/msonet_geovals_rrfs.nc4
```

Comment:
Could we comment this out for now like with the other ob types? We do not have write access to `data/` in RDASApp currently.

Reply:
Yep! I meant to comment that back out and just forgot.